projects/packages/forms/src/modules/file-field/view.js

```diff
@@ -360,6 +359,11 @@ const { state, actions } = store( NAMESPACE, {
 			}
 
 			const file = context.files.find( fileObject => fileObject.id === clientFileId );
+			if ( file && file.url ) {
```

Comment:
is this necessary?

Reply:
URL.revokeObjectURL is a best practice if you are URL.createObjectURL call. Since it unloads the file from memory. As soon as it is not needed anymore.